vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/document/TextDocumentEditor.java

```diff
@@ -45,6 +45,8 @@
 public class TextDocumentEditor implements TextDocumentEventListener, TextDocumentProvider
 {
     private static final Logger LOGGER = LoggerFactory.getLogger(TextDocumentEditor.class);
+    private static final Pattern LINE_SEPARATOR_PATTERN = Pattern.compile("\\R");
```

Comment:
how will it match `\r\n`?

Reply:
Pattern.compile("\\R").matcher("\r\n").matches() -> true